docs/common-server-parameters.md

```diff
@@ -14,13 +14,15 @@ Additional parameters shared by all server-based goals.
 | serverEnvFile | Location of a server environment file to be used by the instance. This replaces the `serverEnv` parameter which is still supported for backwards compatibility.| No |
 | mergeServerEnv | Merge the server environment properties from all specified sources with the default generated `server.env` file in the target server. Conflicts are resolved with the same precedence as the replacement policy when this attribute is set to `false`. The `liberty.env.{var}` Maven properties are highest precedence, followed by the `serverEnvFile` attribute, then the `server.env` file located in the `configDirectory`, and finally the default generated `server.env` file in the target server. The default value is `false`. | No |
 
+#### copying dependencies with liberty-maven-plugin
```

Comment:
I think "Copying" should be capitalized.

Reply:
Got it..thx